02_Instructor_Setup.Rmd

```diff
@@ -0,0 +1,268 @@
+- Create a new Group for each offering of the course.  This can be useful if you have different TAs for every offering and you don't want them to access materials and funding for past and future offerings.
+
+You can always add/remove Groups and Group members later, so we recommend starting out with a single instructor group.  You can decide later if you want to create a new Group for the next course.
+:::
+
+Choose an informative, unique Group name. We suggest a combination of institution-class-role (e.g., `jhu-bio101-instructors`). Only letters, numbers, underscores, and dashes are allowed in Group names.
+
+## Create Terra Group
+
+To create a Group:
+
+::::{.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_create.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+You now have a unique **instructor Group**.
+
+## Add Instructors and TAs to Group
+
+Now that your instructor Group has been created, you should add any additional instructors and TAs. You should also ensure that they have the correct permissions.
+
+:::{.notice}
+Users can be added to a Group as either **Admins** or **Members**.  Admins can add and remove members from the group.
+
+Group roles only affect whether or not someone can add and remove members from the Group.  They do not grant special privileges with respect to Terra resources (Billing Projects and Workspaces).
+
+However, an Admin for the Group can *indirectly* grant access to Terra resources.  By adding someone to the Group, they grant that person access to any Terra resources that the Group has been given permission to access.
+:::
+
+Often it's appropriate to add any **co-instructors as Admins, with TAs as Members**.
+
+To add someone to your group:
+
+:::: {.borrowed_chunk}
+```{r, echo = FALSE, results='asis'}
+cow::borrow_chapter(
+  doc_path = "child/_child_terra_group_add_member.Rmd",
+  repo_name = "jhudsl/AnVIL_Template"
+)
+```
+::::
+
+
+# Set up Terra Billing Projects {#terra-billing-projects}
+
+Terra Billing Projects are how you grant people (including yourself!) permission to compute on AnVIL.  By adding someone to your Billing Project, you enable them to use your funding to carry out activities on AnVIL.
+
+Depending on how you are being funded, this may be taken care of for you. If your funding is being managed by a third party (e.g. through a funding mechanism such as [STRIDES](https://datascience.nih.gov/strides), or through your institution), contact them to determine whether you should create Terra Billing Projects yourself.
+
+The first set of instructions below walk you through creating a Terra Billing Project.  If you do not need to create your own, you can skip down to [adding instructors to Billing Project](#add-instructors-billing-project).
```

Comment:
```suggestion
The first set of instructions below walk you through creating a Terra Billing Project.  **If someone else has provided a billing project**, you can skip down to [adding instructors to Billing Project](#add-instructors-billing-project).
```

Thoughts on rephrasing? Want to make it clear that unless the instructor is in communication with others, they will need to make their own.